Documentation/DocuBlocks/Rest/Graph/general_graph_drop_http_examples.md

```diff
@@ -21,15 +23,46 @@ dropped if they are not used in other graphs.
 
 @RESTRETURNCODE{201}
 Is returned if the graph could be dropped and waitForSync is enabled
-for the `_graphs` collection. 
+for the `_graphs` collection, or given in the request.
 
 @RESTRETURNCODE{202}
-Returned if the graph could be dropped and waitForSync is disabled
-for the `_graphs` collection.
+Is returned if the graph could be dropped and waitForSync is disabled
+for the `_graphs` collection and not given in the request.
+
+@RESTRETURNCODE{403}
+Returned if your user has insufficient rights.
+In order to drop a graph you at least need to have the following privileges:
+  1. `Administrate` access on the Database.
+
+@RESTREPLYBODY{error,boolean,required,}
+Flag if there was an error (true) or not (false)
+It is true in this response.
+
+@RESTREPLYBODY{code,integer,required,}
+The response code.
+
+@RESTREPLYBODY{errorNum,integer,required,}
+ArangoDB error number for the error that occured.
+
+@RESTREPLYBODY{errorMessage,string,required,}
+A message created for this error.
 
 @RESTRETURNCODE{404}
 Returned if no graph with this name could be found.
 
+@RESTREPLYBODY{error,boolean,required,}
```

Comment:
is the last "," intended?

Reply:
it is, the script requires exactly 4 arguments, but the last one is allowed to be empty. ...

Reply:
Ok, thanks. Was expecting something like "optional" for that.